charts/jaeger/README.md

```diff
@@ -264,6 +264,48 @@ helm install jaeger jaegertracing/jaeger \
   --set storage.kafka.topic=<TOPIC>
 ```
 
+## oAuth2 Sidecar
+If extra protection of the Jaeger UI is needed, then the oAuth2 sidecar can be enabled in the Jaeger Query. The oAuth2
+sidecar acts as a security proxy in front of the Jaeger Query service and enforces user authentication before reaching
+the Jaeger UI. This method can work with any valid provider including Keycloak, Azure, Google, GitHub, and more.
+
+Content of the `jaeger-values.yaml` file:
+
+```YAML
+query:
+  enabled: true
+  oAuthSidecar:
+    enabled: true
+    image: quay.io/oauth2-proxy/oauth2-proxy:v7.1.0
+    pullPolicy: IfNotPresent
+    containerPort: 4180
+    args:
+      - --config
+      - /etc/oauth2-proxy/oauth2-proxy.cfg
```

Comment:
this should point to the path of the volumeMount right ? 🤔 
```suggestion
      - /etc/oauth2-proxy
```

Reply:
I believe it should actually point to the data key within the ConfigMap, which is `oauth2-proxy.cfg`. My team currently has the path set as `/etc/oauth2-proxy/oauth2-proxy.cfg` and it is working.

Per the oauth2 docs, the `--config` option takes the path to the config file https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview#command-line-options

Reply:
Yes my bad 🤦🏽‍♂️ . It should be full path to the file. Not sure what i was thinking.